src/lib/updating-element.ts

```diff
@@ -333,12 +333,9 @@ export abstract class UpdatingElement extends HTMLElement {
    *
    *   class MyElement extends LitElement {
    *     static getPropertyDescriptor(name, key, options) {
-   *       const defaultDescriptor = super.getPropertyDescriptor(name, key, options);
```

Comment:
Some unfortunate auto-formatting here... :(

Reply:
fixed. clang-format isn't that smart